dfsclient/src/main/java/org/hpccsystems/dfs/client/HPCCFile.java

```diff
@@ -247,9 +247,13 @@ private void createDataParts() throws HpccFileException
         try
         {
             fileinfoforread = fetchReadFileInfo(fileName, dfuClient, fileAccessExpirySecs, targetfilecluster);
-            if (fileinfoforread.getFileType() != DFUFileType.Flat)
-                throw new Exception("Cannont stream file '" + fileName + "' - File type: '" +  fileinfoforread.getFileType().toString() + "' not supported!");
-
+            if (fileinfoforread.getFileType() != DFUFileType.Flat) {
```

Comment:
format is a little bit off

Reply:
fixed formatting and added backward-compatible constructor to RowServiceinputStream